superagentx_handlers/best_buy.py

```diff
@@ -0,0 +1,63 @@
+from abc import ABC
+import aiohttp
+from superagentx.handler.base import BaseHandler
+from superagentx.handler.decorators import tool
+
+base_url = "https://api.bestbuy.com/v1/products"
+show_options = ("show=customerReviewAverage"
```

Comment:
Make this as `constant` variable, (ie). Upper Case. And align this as per `pep8`

Reply:
changed to uppercase, aligned as per pep8 rule